tools/webpack-core/src/test/RequireMagicCommentsPlugin.test.ts

```diff
@@ -97,7 +97,7 @@ describe("RequireMagicCommentsPlugin", () => {
     expect(fs.readFileSync(path.join(__dirname, "dist/main.js"), "utf8")).to.matchSnapshot();
   });
 
-  it("should work with copyfile comments and addCopyFileSuffix handler", async () => {
+  it.skip("should work with copyfile comments and addCopyFileSuffix handler", async () => {
```

Comment:
@aruniverse is an issue filed?

Reply:
Just created one, will be tracked here: https://github.com/iTwin/itwinjs-backlog/issues/332

Reply:
Nvm, will track here: https://github.com/iTwin/itwinjs-core/issues/4031